pkg/kritis/cryptolib/README.md

```diff
@@ -0,0 +1,41 @@
+# Crypto Library
```

Comment:
Attestation Library (here and elsewhere)

Reply:
Done.